krod/agents/agent_manager.py

```
@@ -0,0 +1,400 @@
+This module identifies query type and selects the appropriate agents and orchestrates the multi-agent workflow.
+
+The module implementats a two-phase approach:
+1. Intent Check: Determines the type of query if it is a follow-up query or needs a new info or both.
+2. Agent Routing: Selects and dispatch appropriate agents based on the intent. 
+"""
+
+
+import os 
+import logging
+import json
+import re
```

Flake8 (7.2.0)

[error] 're' imported but unused

(F401)